chia/_tests/wallet/did_wallet/test_did.py

```diff
@@ -2266,3 +2274,116 @@ async def test_did_coin_records(wallet_environments: WalletTestFramework, monkey
         )
 
     assert len(await wallet.wallet_state_manager.get_spendable_coins_for_wallet(did_wallet.id())) == 1
+
+
+@pytest.mark.limit_consensus_modes(allowed=[ConsensusMode.PLAIN], reason="irrelevant")
+@pytest.mark.anyio
+async def test_alternate_did_recovery(
+    self_hostname: str, simulator_and_wallet: OldSimulatorsAndWallets, monkeypatch: pytest.MonkeyPatch
+):
```

Comment:
Sorry I didn't catch this in the first pass but can we use `WalletTestFramework` here please?

Reply:
actually, this seems copy pasted to some extent, so if it is, I guess it's fine to leave because there would have been a reason we didn't port the original to the framework, I remember there's some limitation to DID wallets that I intended to fix at some point.

Reply:
This is now removed as I test the alternate DIDs now inside all the other existing tests. But github is being weird and my latest commit isn't there yet for some reason.